src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -387,17 +387,39 @@
     type: command
     short-summary: (PREVIEW) Use Azure Dev Spaces with a managed Kubernetes cluster.
     parameters:
+        - name: --update
+          type: bool
+          short-summary: Update to the latest Azure Dev Spaces client components.
         - name: --space -s
           type: string
-          short-summary: Name of the dev space to use.
-        - name: --parent-space -p
-          type: string
-          short-summary: Name of a parent dev space to inherit from when creating a new dev space. By default, if there is already a single dev space with no parent, the new space inherits from this one.
+          short-summary: Name of the new or existing dev space to select. Defaults to an interactive selection experience.
+    examples:
+        - name: Use Azure Dev Spaces with a managed Kubernetes cluster, interactively selecting a dev space.
+          text: |-
+            az aks use-dev-spaces -g my-aks-group -n my-aks
+
```

Comment:
I don't think the blank line here is necessary. We don't do this elsewhere in the _help.py file.

Reply:
Yes. I will remove it.